### ISSUE TITLE
publish website in IIS

### ISSUE THREAD

**Issue by akhtar122** [ytdl-org/youtube-dl#29809]
I have developed a website in Asp.net mvc and trying past my publish the file in www-root under inetpub in c drive but it showing an error  . Can anybody give a solution for this issue
![ErrorInet](https://user-images.githubusercontent.com/68426237/129724322-97a71b6b-2b5d-4247-9f3e-5a12b6a55915.PNG)

**Comment by abdullah-if** [comment]
What this has to do with youtube-dl ?

**Comment by RonRicoSuave** [comment]
In the dirtiest part of your town you will find an old man digging for food in the dumpster behind a restaurant.  Approach him and immediately pull his pants down and begin fellating him without saying anything.  He will tell you the solution to this problem in great detail when you are done.

**Comment by etorresh** [comment]
> In the dirtiest part of your town you will find an old man digging for food in the dumpster behind a restaurant. Approach him and immediately pull his pants down and begin fellating him without saying anything. He will tell you the solution to this problem in great detail when you are done.

I can't reproduce this solution in my testing environments.
I tried a neighborhood with a median household income of $15, 000 and another one with an extremely high crime rate with no luck. Is there any chance you have a tongue piercing or anything else that could affect your technique?

**Comment by Lee-Carre** [comment]
> I have developed a website in Asp.net mvc and trying past my publish the file in www-root under inetpub in c drive but it showing an error . Can anybody give a solution for this issue[?]

Yes; replace that Microsoft proprietary warez with a self-hosted Apache atop a GNU or (Free\|Open\|Net) BSD distro.
